### DISCUSSION TITLE
Fine-tune AraBERT for Semantic Similarity

### DISCUSSION THREAD

**Discussion by Tushar-Faroque** [aub-mind/arabert#160] [Q&A] [answered]
Hello, I am new to this.
Currently working with quranic verses. Any direction on how to fine-tune AraBERT for semantic similarity.

**Comment by WissamAntoun** [comment, marked as answer]
I have done this text before, but I'm sure that if you find something that works for english BERT, it should work directly with AraBERT, you just need to find an equivalent dataset. For dataset check this website https://arbml.github.io/masader/

**Comment by Tushar-Faroque** [comment]
Thanks a lot. I will try it.
